### ISSUE TITLE
biosVfTXTSupport class not supported in imcsdk

### ISSUE THREAD

**Issue by mtimm** [CiscoUcs/imcsdk#179]
When querying the children of the BiosSettings class on a C220-M5 running 3.1(2c) I get an object back from the server that has no matching ManagedObject class.  It is put into a GenericMo object as:

```
GenericMo                       :biosVfTXTSupport
---------
dn                              :sys/rack-unit-1/bios/bios-settings/TXT-Support
rn                              :TXT-Support
vpTXTSupport                    :Disabled
```

**Comment by waghswapnil** [comment]
@mtimm : This class was added in the version 3.1(2b). The current version of the sdk supports 3.1(1d). We need to refresh the sdk to support this class.

**Comment by mtimm** [comment]
Thanks @waghswapnil, so the sdk is lagging by around 8 months or more?  When can it be refreshed?

**Comment by mtimm** [comment]
This was resolved in version 0.9.7.
